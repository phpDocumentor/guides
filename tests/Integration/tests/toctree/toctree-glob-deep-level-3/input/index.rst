==============
Document Title
==============

Lorem Ipsum Dolor.

..  toctree::
    :glob:

    **
