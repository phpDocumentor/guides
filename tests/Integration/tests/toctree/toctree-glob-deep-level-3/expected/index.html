<!-- content start -->
    <div class="section" id="document-title">
            <h1>Document Title</h1>

    <p>Lorem Ipsum Dolor.</p>

            <div class="toc">
    <ul class="menu-level">
    <li class="toc-item">
    <a href="/level-1-1/index.html#level-1-1">Level 1-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-1/index.html#level-2-1">Level 2-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-1/subpage1.html#subpage-1-level-2-1">Subpage 1, Level 2-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-1/subpage2.html#subpage-2-level-2-1">Subpage 2, Level 2-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-2/index.html#level-2-2">Level 2-2</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-2/subpage1.html#subpage-1-level-2-2">Subpage 1, Level 2-2</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/level-2-2/subpage2.html#subpage-1-level-2-2">Subpage 1, Level 2-2</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/subpage1.html#subpage-1-level-1-1">Subpage 1, Level 1-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-1/subpage2.html#subpage-2-level-1-1">Subpage 2, Level 1-1</a>


</li>
    <li class="toc-item">
    <a href="/level-1-2/index.html#level-1-2">Level 1-2</a>


</li>
    <li class="toc-item">
    <a href="/level-1-2/subpage1.html#subpage-1-level-1-2">Subpage 1, Level 1-2</a>


</li>
    <li class="toc-item">
    <a href="/level-1-2/subpage2.html#subpage-1-level-1-2">Subpage 1, Level 1-2</a>


</li>
    <li class="toc-item">
    <a href="/page1.html#page-1">Page 1</a>

        <ul class="section-level-1">
            <li class="toc-item">
    <a href="/page1.html#page-1-level-2">Page 1 Level 2</a>

        <ul class="section-level-2">
            <li class="toc-item">
    <a href="/page1.html#page-1-level-3">Page 1 Level 3</a>

        <ul class="section-level-2">
            <li class="toc-item">
    <a href="/page1.html#page-1-level-4">Page 1 Level 4</a>


</li>
                    </ul>
</li>
                    </ul>
</li>
                    </ul>
</li>
    <li class="toc-item">
    <a href="/page2.html#page-2">Page 2</a>

        <ul class="section-level-1">
            <li class="toc-item">
    <a href="/page2.html#page-2-level-2">Page 2 Level 2</a>

        <ul class="section-level-2">
            <li class="toc-item">
    <a href="/page2.html#page-2-level-3">Page 2 Level 3</a>

        <ul class="section-level-2">
            <li class="toc-item">
    <a href="/page2.html#page-2-level-4">Page 2 Level 4</a>


</li>
                    </ul>
</li>
                    </ul>
</li>
                    </ul>
</li>
    </ul>
</div>
    </div>
<!-- content end -->
